### PR TITLE
Added missing numpy import for resp_to_pandas in Data Extraction docs

### PR DESCRIPTION
**Pull request by sdedkins**
# Citrine Python PR

## Description 
Add missing import statement to code snippet in docs. 

### PR Type:
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

### Adherence to team decisions
- [ ] I have added tests for 100% coverage
- [ ] I have written Numpy-style docstrings for every method and class.
- [ ] I have communicated the downstream consequences of the PR to others.
- [X] I have bumped the version in setup.py

### CODE REVIEW

File: docs/source/data_extraction.rst [hunk 1]
```diff
@@ -150,6 +150,7 @@ or even wrap it in a method that displays multi-row headers:
         import warnings
         from io import StringIO
         import pandas as pd
+        import numpy as np
```
Comment (maxhutch) [inline docs/source/data_extraction.rst:153]:
```suggestion
        import numpy as np
        
```
Suggestion: I think it looks better if we keep the whitespace.

File: src/citrine/__version__.py [hunk 2]
```diff
@@ -1 +1 @@
-__version__ = '1.6.2'
+__version__ = '1.7.1'
```
Comment (maxhutch) [inline src/citrine/__version__.py:1]: Why the jump to 1.7.1?

**Review by maxhutch — CHANGES_REQUESTED** [review]
Nice catch!  The only blocking issue is that the version jump is too big.

File: src/citrine/__version__.py (hunk 2 above)
Comment (sdedkins) [reply src/citrine/__version__.py:1]: I was anticipating merging the my PR with a minor version bump before this one. I anticipated incorrectly :-)

**Review by maxhutch — APPROVED** [review]

**Pull request merged**